.github/ISSUE_TEMPLATE.md

```diff
@@ -1,89 +1,48 @@
+* Operating system: <!-- Replace with the version of your operating system -->
 
-**GitHub Desktop version:** [version here]
-
+## Steps to Reproduce
 <!--
+1. `step 1`
+2. `step 2`
+3. `and so on…`
 
-The operating system you are running on may also help with reproducing the
-issue:
-
- - If you are on macOS, launch `About This Mac` and write down the OS version
-   listed.
- - If you are on Windows, open `Command Prompt` and attach the output of this
-   command: `cmd /c ver`
-
+Include screen shots or gifs if available
 -->
 
-**OS version:** [version here]
-
-### Steps to Reproduce
-
-1. [First step]
-1. [Second step]
-1. [and so on]
-
-<!--
-
-If the issue involves a specific public repository, including the information
-about that repository will make it is easier to recreate the issue.
-
-If you think screenshots or a GIF recording will help demonstrate the issue
-better, feel free to add them here.
-
--->
-
-**Expected behavior:** [What you expected to happen]
-
-**Actual behavior:** [What actually happened]
-
-**Reproduces how often:** [What percentage of the time does it reproduce?]
-
+### Expected Behavior
+<!-- Replace with what you expected to happen -->
+### Actual Behavior
+<!-- Replace with what actually happens -->
+### How often does this issue occur?
+<!-- Sometimes`, `Most of the time`, or `Every time -->
+## Additional Information
+<!-- 
+Replace with any additional information, configuration, or data that might be necessary to reproduce the issue.
+
+If you are dealing with a performance issue or regression, attaching a Timeline profile of the task will help the developers understand the runtime behavior of the application on your machine.
+	* https://github.com/desktop/desktop/blob/master/docs/contributing/timeline-profile.md
+ -->
 ### Logs
+<!-- 
+Replace with your log file (You can simply drag your file here to insert it) 
 
-<!--
-
-There may be some relevant information in log files generated by GitHub
-Desktop:
-
- - If you are on macOS, attach the most recent log file from:
-   `~/Library/Application Support/GitHub Desktop/logs/*.desktop.production.log`
- - If you are on Windows, attach the most recent log file from:
-   `%APPDATA%\GitHub Desktop\logs\*.desktop.production.log`
-
-The log files are organized by date, so see if anything was generated for
-today's date.
-
--->
-
-#### Additional Information
-
-<!--
-
-Any additional information, configuration or data that might be necessary to
-reproduce the issue.
-
-If you are dealing with a performance issue or regression, attaching a
-[Timeline profile](https://github.com/desktop/desktop/blob/master/docs/contributing/timeline-profile.md)
-of the task will help the developers understand the runtime behavior of the
-application on your machine.
+macOS logs location: `~/Library/Application Support/GitHub Desktop/logs/*.desktop.production.log`
+Windows logs location: `%APPDATA%\GitHub Desktop\logs\*.desktop.production.log`
```

Comment:
Might be a feature to consider for the future that will zip up today's log files and drop them on the OS desktop to be more easily dragged?